tap_mssql/sync_strategies/common.py

```diff
@@ -98,6 +100,7 @@ def row_to_singer_record(catalog_entry, version, row, columns, time_extracted, c
     use_date_data_type_format = config.get("use_date_datatype") or default_date_format()
     for idx, elem in enumerate(row):
         property_type = catalog_entry.schema.properties[columns[idx]].type
+        property_format = catalog_entry.schema.properties[columns[idx]].format

```

Comment:
property_format may be set on a number of record types in the schema, but it is required to be extracted to check for `singer.decimal`